docs/unlocked_package.rst

```diff
@@ -26,18 +26,18 @@ To create a new unlocked package version, run the ``create_package_version`` tas
 
     $ cci task run create_package_version --org <org_name> --package_type Unlocked
 
-This task looks in your default DevHub org (as configured in ``sfdx``) for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of the ``cumulusci.yml`` file). If a matching package doesn't exist, the task then submits a request to create the package version. When completed (which can take some time), the task outputs a ``SubscriberPackageVersion Id``, which can be used to install the package in another org.
+This task looks in your default DevHub org (as configured in ``sfdx``) for an unlocked package with the name and namespace specified in the task options (defaulting to the name and namespace from the ``project__package`` section of the ``cumulusci.yml`` file). If a matching package doesn't exist, the task then submits a request to create the package version. When completed (which can take some time), the task outputs a ``SubscriberPackageVersion`` id, which can be used to install the package in another org.
 
-If a package version already exists with the exact same contents, its ``SubscriberPackageVersion Id`` is returned.
+If a package version already exists with the exact same contents, its ``SubscriberPackageVersion`` id is returned.
```

Comment:
```suggestion
If a package version already exists with the exact same contents, as determined by the hash of the package metadata, its ``SubscriberPackageVersion`` id is returned rather than creating a new version.
```